src/components/list.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import Form from './Form';
+
+export default function List() {
+  const [books] = useState([
+    {
+      id: 1,
+      title: 'Romoe and Juliet',
+      author: 'William Shakespear',
+      category: 'Non-fiction',
+      completed: false,
+
+    },
+    {
+      id: 1,
+      title: 'Art of hustling',
+      author: 'Dj sbu',
+      category: 'Non-fiction',
+      completed: false,
+
+    },
+  ]);
+
+  return (
+    <div>
+      {books.map((book) => (
+        <Form
+          key={book.id}
+          list={book}
+        />
+      ))}
+
+    </div>
+  );
+}
```

Comment:
- [ ] Good job so far @techmoves There are just a few changes you need to make and you'll be on your way to the next milestone :+1: 
- [ ] The first thing I noticed is that per the project requirements, you need to create a `<Books>` component as the main entry point. However, in your current implementation, you have a `<List>` component serving this purpose. To align with the requirements, you should consider renaming the `<List>` component to `<Books>` :100: 
- Additionally, I will suggest you create a separate component for `<BookList>` and `<Book>` as child components of `<Books>`. :+1: 
- This will help improve code organization and component reusability. :100: 

- [ ] The `<Books>` component should render the list of books using the `<BookList>` component. However, in your code, you are rendering `<Form>` components instead. Kindly modify the rendering logic to use the `<BookList> ` component and pass the books' data as props. :+1: 

- [ ] After this, you can create a new component called `<Book>` that takes book data as a `prop` and renders the title and author information. Currently, the `<Form>` component is being used for this purpose, which doesn't align with the project requirements so kindly change this as well. :+1: